lib/overload.pm

```diff
@@ -1239,7 +1243,7 @@ Put this in F<symbolic.pm> in your Perl library directory:
 
 This module is very unusual as overloaded modules go: it does not
 provide any usual overloaded operators, instead it provides an
-implementation for C<L</nomethod>>.  In this example the C<nomethod>
+implementation for L</C<nomethod>>.  In this example the C<nomethod>
```

Comment:
Why this change?  It's my experience that the original method looks better for internal links; the changed-to form for external links.  Sadly.

Reply:
That's an oversight. I have no opinion on POD style personally. I will remove it.